manga_notify/drivers/basic_rss.py

```diff
@@ -0,0 +1,63 @@
+import typing
+import aiohttp
+
+import xml.etree.ElementTree as ET
+
+from . import driver
+from . import common_message
+from ..channels import channel
+from ..database import feed_storage
+
+
+UA = (
+    'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:93.0) '
+    'Gecko/20100101 Firefox/93.0'
+)
+
+
+def iter_items(items):
+    for item in items:
+        title = item.find('title').text
+        link = item.find('link').text
+        if not title:
+            continue
+        yield common_message.ParsingItem(
+            name=title,
+            link=link,
+        )
+
+
+class BasicRss(driver.Driver):
+    def is_match(self, url: str) -> bool:
+        return NotImplementedError
```

Comment:
NotImplementedError is an exception. You should use `rise` instead of `return`.

Reply:
Sorry, fixed